arc/portal_gallery_package/DeploymentTemplates/CreateUiDefinition.json

```diff
@@ -0,0 +1,107 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+    "handler": "Microsoft.Azure.CreateUIDef",
+    "version": "0.1.2-preview",
+    "parameters": {
+        "resourceTypes": ["Microsoft.KubernetesConfiguration/extensions"],
+        "config": {
+            "isWizard": true,
+            "basics": {
+                "subscription": {
+                    "resourceProviders": [
+                        "microsoft.kubernetes/connectedclusters"
+                    ]
+                },
+                "location": {
+                    "label": "Region",
+                    "resourceTypes": [
+                        "microsoft.kubernetes/connectedclusters"
+                    ],
+                    "visible": true
+                },
+                "resourceGroup": {
+                  "allowExisting": true
+              }
+            }
+        },
+        "basics": [ 
+            {
+                "name": "connectedCluster",
+                "type": "Microsoft.Solutions.ResourceSelector",
+                "label": "Connected cluster",
+                "resourceType": "microsoft.kubernetes/connectedclusters",
+                "options": {
+                    "filter": {
+                        "subscription": "onBasics",
+                        "location": "onBasics"
+                    }
+                }
+            },
+            {
+                "name": "extensionName",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Extension name",
+                "multiLine": false,
+                "defaultValue": "akvsecretsprovider",
+                "constraints": {
+                  "required": true
+                },
+                "visible": true
+            }            
+        ],
+      "steps": [                 
+        {
+          "name": "configuration",
+          "label": "Configuration",
+          "elements": [
+            {
+              "name": "enableSecretRotation",
+              "type": "Microsoft.Common.CheckBox",
+              "label": "Enable secret rotation"
+            },
+            {
+                "name": "rotationPollInterval",
+                "type": "Microsoft.Common.Slider",
+                "min": 2,
```

Comment:
I think setting minimum of `2m` is ok for now but in the future if there is a request for setting it in the range of few seconds, how would we support it? Would that be breaking change to allow `subLabel: s`?

Reply:
This will not be a breaking change. We can change it to `s` down the line if needed. This gets converted to string and passed as an input param for ARM deployment template.

```json
"rotationPollInterval": {
    "defaultValue": "2m",
    "type": "String"
 }
```